src/index.js

```diff
@@ -68,6 +94,16 @@ function is(x, type) {
     return false
 }
 
+/**
+ * Often, we don't really care whether the object is a certain type,
+ * we just want to know whether we can treat it like a certain type.
+ * Use like(subject, type) to test whether a subject adheres to the same interface as is defined by type
+ * Example: like(looker, 'object')
+ *
+ * @param {object|function} x
+ * @param {string|function} type
```

Comment:
Maybe just make it `@param {function} type`. With string it will sorta work, but it does not make much sense (what does it mean for x to be 'like a function' or 'like an object'?)